2025-01-31 22:08:13,375:DEBUG:certbot._internal.main:certbot version: 3.1.0
2025-01-31 22:08:13,375:DEBUG:certbot._internal.main:Location of certbot entry point: /usr/local/bin/certbot
2025-01-31 22:08:13,375:DEBUG:certbot._internal.main:Arguments: ['--webroot', '--webroot-path=/var/www/certbot', '--email', 'karimmaroous@gmail.com', '--agree-tos', '--no-eff-email', '--staging', '-d', 'opsforall.net']
2025-01-31 22:08:13,375:DEBUG:certbot._internal.main:Discovered plugins: PluginsRegistry(PluginEntryPoint#manual,PluginEntryPoint#null,PluginEntryPoint#standalone,PluginEntryPoint#webroot)
2025-01-31 22:08:13,388:DEBUG:certbot._internal.log:Root logging level set at 30
2025-01-31 22:08:13,389:DEBUG:certbot._internal.plugins.selection:Requested authenticator webroot and installer None
2025-01-31 22:08:13,389:DEBUG:certbot._internal.plugins.selection:Single candidate plugin: * webroot
Description: Saves the necessary validation files to a .well-known/acme-challenge/ directory within the nominated webroot path. A separate HTTP server must be running and serving files from the webroot path. HTTP challenge only (wildcards not supported).
Interfaces: Authenticator, Plugin
Entry point: EntryPoint(name='webroot', value='certbot._internal.plugins.webroot:Authenticator', group='certbot.plugins')
Initialized: <certbot._internal.plugins.webroot.Authenticator object at 0x7558f21b3950>
Prep: True
2025-01-31 22:08:13,390:DEBUG:certbot._internal.plugins.selection:Selected authenticator <certbot._internal.plugins.webroot.Authenticator object at 0x7558f21b3950> and installer None
2025-01-31 22:08:13,390:INFO:certbot._internal.plugins.selection:Plugins selected: Authenticator webroot, Installer None
2025-01-31 22:08:13,509:DEBUG:certbot._internal.main:Picked account: <Account(RegistrationResource(body=Registration(key=None, contact=(), agreement=None, status=None, terms_of_service_agreed=None, only_return_existing=None, external_account_binding=None), uri='https://acme-staging-v02.api.letsencrypt.org/acme/acct/182784314', new_authzr_uri=None, terms_of_service=None), 5c0ea7419c19db18f607881e60bb62df, Meta(creation_dt=datetime.datetime(2025, 1, 31, 21, 18, 24, tzinfo=datetime.timezone.utc), creation_host='f5a16ff59e49', register_to_eff=None))>
2025-01-31 22:08:13,511:DEBUG:acme.client:Sending GET request to https://acme-staging-v02.api.letsencrypt.org/directory.
2025-01-31 22:08:13,515:DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): acme-staging-v02.api.letsencrypt.org:443
2025-01-31 22:08:13,691:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "GET /directory HTTP/1.1" 200 974
2025-01-31 22:08:13,692:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 31 Jan 2025 22:08:13 GMT
Content-Type: application/json
Content-Length: 974
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "YXpwvw8Ohig": "https://community.letsencrypt.org/t/adding-random-entries-to-the-directory/33417",
  "keyChange": "https://acme-staging-v02.api.letsencrypt.org/acme/key-change",
  "meta": {
    "caaIdentities": [
      "letsencrypt.org"
    ],
    "profiles": {
      "classic": "The same profile you're accustomed to",
      "tlsserver": "https://letsencrypt.org/2025/01/09/acme-profiles/"
    },
    "termsOfService": "https://letsencrypt.org/documents/LE-SA-v1.4-April-3-2024.pdf",
    "website": "https://letsencrypt.org/docs/staging-environment/"
  },
  "newAccount": "https://acme-staging-v02.api.letsencrypt.org/acme/new-acct",
  "newNonce": "https://acme-staging-v02.api.letsencrypt.org/acme/new-nonce",
  "newOrder": "https://acme-staging-v02.api.letsencrypt.org/acme/new-order",
  "renewalInfo": "https://acme-staging-v02.api.letsencrypt.org/draft-ietf-acme-ari-03/renewalInfo",
  "revokeCert": "https://acme-staging-v02.api.letsencrypt.org/acme/revoke-cert"
}
2025-01-31 22:08:13,693:DEBUG:certbot._internal.display.obj:Notifying user: Requesting a certificate for opsforall.net
2025-01-31 22:08:13,697:DEBUG:acme.client:Requesting fresh nonce
2025-01-31 22:08:13,699:DEBUG:acme.client:Sending HEAD request to https://acme-staging-v02.api.letsencrypt.org/acme/new-nonce.
2025-01-31 22:08:13,759:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "HEAD /acme/new-nonce HTTP/1.1" 200 0
2025-01-31 22:08:13,764:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 31 Jan 2025 22:08:13 GMT
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: xMYr_xat4dzRU-tWy6QTEEoZCCqv8CbsZWn1WW-MI4RD-TVE5II
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800


2025-01-31 22:08:13,765:DEBUG:acme.client:Storing nonce: xMYr_xat4dzRU-tWy6QTEEoZCCqv8CbsZWn1WW-MI4RD-TVE5II
2025-01-31 22:08:13,765:DEBUG:acme.client:JWS payload:
b'{\n  "identifiers": [\n    {\n      "type": "dns",\n      "value": "opsforall.net"\n    }\n  ]\n}'
2025-01-31 22:08:13,771:DEBUG:acme.client:Sending POST request to https://acme-staging-v02.api.letsencrypt.org/acme/new-order:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS1zdGFnaW5nLXYwMi5hcGkubGV0c2VuY3J5cHQub3JnL2FjbWUvYWNjdC8xODI3ODQzMTQiLCAibm9uY2UiOiAieE1Zcl94YXQ0ZHpSVS10V3k2UVRFRW9aQ0NxdjhDYnNaV24xV1ctTUk0UkQtVFZFNUlJIiwgInVybCI6ICJodHRwczovL2FjbWUtc3RhZ2luZy12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL25ldy1vcmRlciJ9",
  "signature": "2osfT12SBzQTevb8eALK6p32rKi0-wsc38fcM0JJDReNs0VeHWs2j2zmh9K5CzuA5CR67FJ2LX_qd5ZTb2UYbnrv3-HUMKRZ5WEmvxP5KaBGzck6F2ikZ9-edZz9GIIfX4nYzA9u2qoQZ0anjuvQNACkxIyNNGc5hjUSxmpLkeQkQITPufSn2AItzauLEXbTuNoKdjAZ2DYTvW6I9_OWcLW4J10MXfygL4lJFYmj-KeDvC43cQN5raVh3s7tFO5szLgj7QE2gVVrJTLg1MouUcE1CJnrU5KOc8K0jOPSBBkYjyZWtDiLh0Jo231Fu-zlA3s9tzK0BGA3z29U-6orZw",
  "payload": "ewogICJpZGVudGlmaWVycyI6IFsKICAgIHsKICAgICAgInR5cGUiOiAiZG5zIiwKICAgICAgInZhbHVlIjogIm9wc2ZvcmFsbC5uZXQiCiAgICB9CiAgXQp9"
}
2025-01-31 22:08:13,868:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "POST /acme/new-order HTTP/1.1" 201 359
2025-01-31 22:08:13,870:DEBUG:acme.client:Received response:
HTTP 201
Server: nginx
Date: Fri, 31 Jan 2025 22:08:13 GMT
Content-Type: application/json
Content-Length: 359
Connection: keep-alive
Boulder-Requester: 182784314
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index"
Location: https://acme-staging-v02.api.letsencrypt.org/acme/order/182784314/22333869044
Replay-Nonce: N9Ep_yiT5tCeBExB6ueMb7TTJ-U9RM2ANunsOPUNZWXV54afXLA
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "status": "pending",
  "expires": "2025-02-07T22:08:13Z",
  "identifiers": [
    {
      "type": "dns",
      "value": "opsforall.net"
    }
  ],
  "authorizations": [
    "https://acme-staging-v02.api.letsencrypt.org/acme/authz/182784314/15902508214"
  ],
  "finalize": "https://acme-staging-v02.api.letsencrypt.org/acme/finalize/182784314/22333869044"
}
2025-01-31 22:08:13,871:DEBUG:acme.client:Storing nonce: N9Ep_yiT5tCeBExB6ueMb7TTJ-U9RM2ANunsOPUNZWXV54afXLA
2025-01-31 22:08:13,871:DEBUG:acme.client:JWS payload:
b''
2025-01-31 22:08:13,873:DEBUG:acme.client:Sending POST request to https://acme-staging-v02.api.letsencrypt.org/acme/authz/182784314/15902508214:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS1zdGFnaW5nLXYwMi5hcGkubGV0c2VuY3J5cHQub3JnL2FjbWUvYWNjdC8xODI3ODQzMTQiLCAibm9uY2UiOiAiTjlFcF95aVQ1dENlQkV4QjZ1ZU1iN1RUSi1VOVJNMkFOdW5zT1BVTlpXWFY1NGFmWExBIiwgInVybCI6ICJodHRwczovL2FjbWUtc3RhZ2luZy12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LzE4Mjc4NDMxNC8xNTkwMjUwODIxNCJ9",
  "signature": "oRJN78fK-yiQW6Vx8rY9EsNuYXaJFMfnZXNg-gd2OrjZ7YOoW-xWfxpH-3ze5TckPAHI0HJBpzX8sCyGWP5nzRLrBW5xN0SFg-R69reRH9hQtA51n9prYqLn2KMuWGPmg5dWqrsPzu_8FcFx6k_JAj7ArNityvydFcgGUfhdxxpNsGIstBAH0w_UbptEyqLKpxEcW-4d0qlzW1aI1l7ZrI8jhtroGze09O-DFvFfoSPMF4gOjr8yf7CF7PHu7tmZrSuRJFkrxbMEIUO_Qopj4AJMGuKD4IzovS11nwXZRe0B5gIzrTZ7Z3pMBpyJNem2607gZND3eywL9NZznPUPfg",
  "payload": ""
}
2025-01-31 22:08:13,940:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "POST /acme/authz/182784314/15902508214 HTTP/1.1" 200 839
2025-01-31 22:08:13,940:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 31 Jan 2025 22:08:13 GMT
Content-Type: application/json
Content-Length: 839
Connection: keep-alive
Boulder-Requester: 182784314
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: N9Ep_yiThg_JB-RKsG1gC19hSvH2VRBPYyDINmVJ3piCdJO3uUk
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "opsforall.net"
  },
  "status": "pending",
  "expires": "2025-02-07T22:08:13Z",
  "challenges": [
    {
      "type": "dns-01",
      "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902508214/bKFLMw",
      "status": "pending",
      "token": "gpg_yczg3blNHQE-PfcJ37Emd9pZMmu_RYPxNQjIjPI"
    },
    {
      "type": "http-01",
      "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902508214/e5Tm4Q",
      "status": "pending",
      "token": "gpg_yczg3blNHQE-PfcJ37Emd9pZMmu_RYPxNQjIjPI"
    },
    {
      "type": "tls-alpn-01",
      "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902508214/_iSroQ",
      "status": "pending",
      "token": "gpg_yczg3blNHQE-PfcJ37Emd9pZMmu_RYPxNQjIjPI"
    }
  ]
}
2025-01-31 22:08:13,940:DEBUG:acme.client:Storing nonce: N9Ep_yiThg_JB-RKsG1gC19hSvH2VRBPYyDINmVJ3piCdJO3uUk
2025-01-31 22:08:13,941:INFO:certbot._internal.auth_handler:Performing the following challenges:
2025-01-31 22:08:13,941:INFO:certbot._internal.auth_handler:http-01 challenge for opsforall.net
2025-01-31 22:08:13,941:INFO:certbot._internal.plugins.webroot:Using the webroot path /var/www/certbot for all unmatched domains.
2025-01-31 22:08:13,941:DEBUG:certbot._internal.plugins.webroot:Creating root challenges validation dir at /var/www/certbot/.well-known/acme-challenge
2025-01-31 22:08:13,946:DEBUG:certbot._internal.plugins.webroot:Attempting to save validation to /var/www/certbot/.well-known/acme-challenge/gpg_yczg3blNHQE-PfcJ37Emd9pZMmu_RYPxNQjIjPI
2025-01-31 22:08:13,949:DEBUG:acme.client:JWS payload:
b'{}'
2025-01-31 22:08:13,952:DEBUG:acme.client:Sending POST request to https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902508214/e5Tm4Q:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS1zdGFnaW5nLXYwMi5hcGkubGV0c2VuY3J5cHQub3JnL2FjbWUvYWNjdC8xODI3ODQzMTQiLCAibm9uY2UiOiAiTjlFcF95aVRoZ19KQi1SS3NHMWdDMTloU3ZIMlZSQlBZeURJTm1WSjNwaUNkSk8zdVVrIiwgInVybCI6ICJodHRwczovL2FjbWUtc3RhZ2luZy12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2NoYWxsLzE4Mjc4NDMxNC8xNTkwMjUwODIxNC9lNVRtNFEifQ",
  "signature": "eUyCMV03QDVcuTJjGFMlRD2XvRG5wJNbwobfpMlcDqowhMEFP281NYFjjSBU7ujwfgF-JEWA1dYsVEdHeRO97HGTYoAfwR0Uq5Si4iztgbNcB-q5kHkiC3efGpNuB98vCvIGcrDi4kHrvnB3PCXHQU3nUP_5uqgArl2KuHibbFSHWP6ANB-E4fWnTYdRym04Fpl_KXYUzH8LcoB_KhYB6N1OcJe0jK-US-nhAqo3XHm_8b66o53tB_r-rKHC78YBLrrSm2okay2Nuf5Z3fYQ48HgUxiIXrn1M7WNnBLbcrzHb5zGhzpuR2GqJ2Ol8NSKaPBRNva11-CZrF_CPNzgUw",
  "payload": "e30"
}
2025-01-31 22:08:14,020:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "POST /acme/chall/182784314/15902508214/e5Tm4Q HTTP/1.1" 200 201
2025-01-31 22:08:14,021:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 31 Jan 2025 22:08:13 GMT
Content-Type: application/json
Content-Length: 201
Connection: keep-alive
Boulder-Requester: 182784314
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index", <https://acme-staging-v02.api.letsencrypt.org/acme/authz/182784314/15902508214>;rel="up"
Location: https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902508214/e5Tm4Q
Replay-Nonce: N9Ep_yiTTQizht3GFJ_77CGIQwMDcd85xs0eKNdLEIISAHM7bLo
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "type": "http-01",
  "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902508214/e5Tm4Q",
  "status": "pending",
  "token": "gpg_yczg3blNHQE-PfcJ37Emd9pZMmu_RYPxNQjIjPI"
}
2025-01-31 22:08:14,022:DEBUG:acme.client:Storing nonce: N9Ep_yiTTQizht3GFJ_77CGIQwMDcd85xs0eKNdLEIISAHM7bLo
2025-01-31 22:08:14,022:INFO:certbot._internal.auth_handler:Waiting for verification...
2025-01-31 22:08:15,023:DEBUG:acme.client:JWS payload:
b''
2025-01-31 22:08:15,028:DEBUG:acme.client:Sending POST request to https://acme-staging-v02.api.letsencrypt.org/acme/authz/182784314/15902508214:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS1zdGFnaW5nLXYwMi5hcGkubGV0c2VuY3J5cHQub3JnL2FjbWUvYWNjdC8xODI3ODQzMTQiLCAibm9uY2UiOiAiTjlFcF95aVRUUWl6aHQzR0ZKXzc3Q0dJUXdNRGNkODV4czBlS05kTEVJSVNBSE03YkxvIiwgInVybCI6ICJodHRwczovL2FjbWUtc3RhZ2luZy12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LzE4Mjc4NDMxNC8xNTkwMjUwODIxNCJ9",
  "signature": "3-nRr7-MRuIgdTUuZHKHQsSsem37xrv3hQ1HZ-6onFhAZi3a9jeAeGH55NnVOlDzTmAEPcNYvjOeymI2_F6wJ0DORwnn-ed5UrNvLBAWfzdhuOA3TM_tAliAEVp2nK6TW5BmZ0s6XdNFHeRREniyhA-nRvXniu5I64otkB3xZGZCYq71hK3q8RhBv9yh2YvVyxKJRTnDm714STHdpA8VuwjJarz1sNCZ1GUf3Bdj28zcn-U7LPMqgyjG47IcjgWFaZkQWbDQQbJWkEQyMRXT-BlT12Ik3e8k9iCfj5ejlBfEg0fqvIMtt1pA3tZ-ANGTTm-nGskLmtO0ABevMO4XJg",
  "payload": ""
}
2025-01-31 22:08:15,097:DEBUG:urllib3.connectionpool:https://acme-staging-v02.api.letsencrypt.org:443 "POST /acme/authz/182784314/15902508214 HTTP/1.1" 200 1031
2025-01-31 22:08:15,098:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 31 Jan 2025 22:08:15 GMT
Content-Type: application/json
Content-Length: 1031
Connection: keep-alive
Boulder-Requester: 182784314
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-staging-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: xMYr_xatGu6fGVOraM6iv-F0gcsFofZqB1Xi4gglzvRgfXnHll4
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "opsforall.net"
  },
  "status": "invalid",
  "expires": "2025-02-07T22:08:13Z",
  "challenges": [
    {
      "type": "http-01",
      "url": "https://acme-staging-v02.api.letsencrypt.org/acme/chall/182784314/15902508214/e5Tm4Q",
      "status": "invalid",
      "validated": "2025-01-31T22:08:13Z",
      "error": {
        "type": "urn:ietf:params:acme:error:connection",
        "detail": "54.226.228.6: Fetching http://opsforall.net/.well-known/acme-challenge/gpg_yczg3blNHQE-PfcJ37Emd9pZMmu_RYPxNQjIjPI: Connection refused",
        "status": 400
      },
      "token": "gpg_yczg3blNHQE-PfcJ37Emd9pZMmu_RYPxNQjIjPI",
      "validationRecord": [
        {
          "url": "http://opsforall.net/.well-known/acme-challenge/gpg_yczg3blNHQE-PfcJ37Emd9pZMmu_RYPxNQjIjPI",
          "hostname": "opsforall.net",
          "port": "80",
          "addressesResolved": [
            "54.226.228.6"
          ],
          "addressUsed": "54.226.228.6"
        }
      ]
    }
  ]
}
2025-01-31 22:08:15,098:DEBUG:acme.client:Storing nonce: xMYr_xatGu6fGVOraM6iv-F0gcsFofZqB1Xi4gglzvRgfXnHll4
2025-01-31 22:08:15,098:INFO:certbot._internal.auth_handler:Challenge failed for domain opsforall.net
2025-01-31 22:08:15,098:INFO:certbot._internal.auth_handler:http-01 challenge for opsforall.net
2025-01-31 22:08:15,098:DEBUG:certbot._internal.display.obj:Notifying user: 
Certbot failed to authenticate some domains (authenticator: webroot). The Certificate Authority reported these problems:
  Domain: opsforall.net
  Type:   connection
  Detail: 54.226.228.6: Fetching http://opsforall.net/.well-known/acme-challenge/gpg_yczg3blNHQE-PfcJ37Emd9pZMmu_RYPxNQjIjPI: Connection refused

Hint: The Certificate Authority failed to download the temporary challenge files created by Certbot. Ensure that the listed domains serve their content from the provided --webroot-path/-w and that files created there can be downloaded from the internet.

2025-01-31 22:08:15,101:DEBUG:certbot._internal.error_handler:Encountered exception:
Traceback (most recent call last):
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.

2025-01-31 22:08:15,102:DEBUG:certbot._internal.error_handler:Calling registered functions
2025-01-31 22:08:15,102:INFO:certbot._internal.auth_handler:Cleaning up challenges
2025-01-31 22:08:15,102:DEBUG:certbot._internal.plugins.webroot:Removing /var/www/certbot/.well-known/acme-challenge/gpg_yczg3blNHQE-PfcJ37Emd9pZMmu_RYPxNQjIjPI
2025-01-31 22:08:15,106:DEBUG:certbot._internal.plugins.webroot:All challenges cleaned up
2025-01-31 22:08:15,106:DEBUG:certbot._internal.log:Exiting abnormally:
Traceback (most recent call last):
  File "/usr/local/bin/certbot", line 33, in <module>
    sys.exit(load_entry_point('certbot', 'console_scripts', 'certbot')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/main.py", line 19, in main
    return internal_main.main(cli_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1873, in main
    return config.func(config, plugins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1579, in certonly
    lineage = _get_and_save_cert(le_client, config, domains, certname, lineage)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 142, in _get_and_save_cert
    lineage = le_client.obtain_and_enroll_certificate(domains, certname)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 518, in obtain_and_enroll_certificate
    cert, chain, key, _ = self.obtain_certificate(domains)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 429, in obtain_certificate
    orderr = self._get_order_and_authorizations(csr.data, self.config.allow_subset_of_names)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 497, in _get_order_and_authorizations
    authzr = self.auth_handler.handle_authorizations(orderr, self.config, best_effort)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.
2025-01-31 22:08:15,112:ERROR:certbot._internal.log:Some challenges have failed.
